terraform/modules/benchmark/service/benchmark_base.rb

```diff
@@ -1,5 +1,26 @@
 # frozen_string_literal: true
 
+def cleanup(db_created)
+  # Delete downloaded snapshot if it exists.
+  FileUtils.rm_f(@snapshot_path) unless !@snapshot_downloaded || @snapshot_path.nil?
+  FileUtils.rm_rf("#{WORKING_DIR}/snapshot_dl_files")
```

Comment:
Probably out of scope for this PR, but are we doing this kind of cleanup in case of a successful run?

Reply:
Added this to #137